pkg/synthesis/synthesis_test.go

```diff
@@ -39,57 +38,57 @@ const (
 type synthesisTest struct {
 	name                  string
 	exData                tests.ExampleSynthesis
-	allowOnlyFromCategory dfw.DfwCategory
+	allowOnlyFromCategory collector.DfwCategory
 	noHint                bool // run also with no hint
 }
 
```

Comment:
```suggestion
var minCategory = collector.EthernetCategory
```

or add function `MinCategory()` from where categories are defined